pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -945,13 +1002,14 @@ def setSymbolBrush(self, *args, **kwargs):
 
     def setSymbolSize(self, size: int):
```

Comment:
oh, this should be updated to be `size: int | list[int]`  ...not your change, but a mistake I made.

Reply:
There is some air of uncertainty on whether list includes np.ndarray, or if and when that should be array_like.

I have no suggested answer to that question.